MvvmCross/Binding/MvxCoreBindingBuilder.cs

```diff
@@ -155,7 +156,7 @@ protected virtual void FillValueCombiners(IMvxValueCombinerRegistry registry)
             registry.AddOrOverwrite("And", new MvxAndValueCombiner());
             registry.AddOrOverwrite("Or", new MvxOrValueCombiner());
             registry.AddOrOverwrite("XOr", new MvxXorValueCombiner());
-			registry.AddOrOverwrite("Inverted", new MvxInvertedValueCombiner());
+            registry.AddOrOverwrite("Inverted", new MvxInvertedValueCombiner());
```

Comment:
There is already a `Not` value combiner. No need for `Inverted` as it does the same.

Reply:
This change was for the format of the line, it was using tabs vs the rest of the file using spaces. The feature was added in PR #1828. 

I'm actual going to close this PR as it is not needed, I found out you can use the `ByCombining` Fluent API to achieve this behaviour.

```csharp
bindingSet.Bind(label)
    .For(v => v.BindVisible())
    .ByCombining(new MvxInvertedValueCombiner(), vm => vm.IsDisabled);
```

I can submit a new PR if you want to fix the formatting on the `MvxCoreBindingBuilder.cs` file?